.github/workflows/zizmor.yml

```
@@ -0,0 +1,31 @@
+name: GitHub Actions Security Analysis with zizmor 🌈
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["*"]
```

```suggestion
    branches: ["**"]
```
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
> `*`: Matches zero or more characters, but does not match the `/` character. For example, `Octo*` matches `Octocat`.
> `**`: Matches zero or more of any character.